assets/scss/base/form/_hint.scss

```diff
@@ -73,11 +73,15 @@ List
 
 A Form hint list. The form hint list can also be used in conjunction with `formHintHelper.js` for instance if you
 had a password text input and wanted to check the hints have been correctly matched.
+Optionally wrap the `.form-hint-list-item__indicator` element in a `.form-hint-list-item__primary` element to give a
```

Comment:
Wording wise - should this be optional? If you are using indicator then you "should" always have this as you may not have control over the amount of text used or when the browser width takes it onto two lines.